### PR TITLE
Self Surgery

### PR DESCRIPTION
**Pull request by BetaCentari**
Push for enabling all self surgery, and slightly modifying items in Nanomed Vendor.

## About The Pull Request

This PR  makes all surgeries self performable along with a very minor change in adding more spray bottles to the NanoMed Plus vendor. 

## Why It's Good For The Game

It  gives the more gutsy doctors the ability to perform any surgery they'd like on themselves. And a minor change in line with the development of new SOP to allow more members of the medical department to keep the place clean. 

## Changelog
:cl:
tweak: Surgeries are now self performable.
tweak: Nanomed Vendors now sell 3 Spray Bottles instead of 1. 
/:cl:

### CODE REVIEW

**Comment by Karmic-Skink** [comment]
So essentially we are removing jobs for medical players?

**Comment by BetaCentari** [comment]
> So essentially we are removing jobs for medical players?

I've barely seen anyone use implants since the removal of the Autodoc and not many players are willing to sit unconscious for long periods of time while medical does surgeries, replaces organs, and installs implants. Most of my playtime is in medical and I never see implants get used or want to get used anymore since AutoDocs got removed from upstream.

**Comment by bluey89962** [comment]
If I'm being honest, while I did dislike the removal of the autodoc at first, I really would rather not have it back now. 

It just incentivises people to walk into medbay storage, print a bunch of implants, and install all of them for no good reason other than being a gamer. Does a cook really need x-ray eyes and a jetpack?

Implants are still around, and doctors are also still around. All you need to do is walk into medbay, ask them to install an implant for you. If they are not willing to wait for a few minutes to get their upgrades, they can just not use them. The lack of use now actually shows that they were never a necessary thing for all of those people who broke into medbay to use autodoc - they did it just because they wanted the upgrades for the sake of being upgraded.

For self surgery, I honestly don't have a formed opinion on it yet, though I feel like it should definitely be job restricted to the CMO and _maybe_ docs and paras.

t. like one of the few other CMO mains aswell

**Comment by williamcll** [comment]
> 

No their job is to man the gauss

**Comment by covertcorvid** [comment]
Any chance we can try just the self-surgery improvement without the autodoc? It does technically let people implant themselves, but might make them think twice about not getting a doctor to do it

**Comment by BetaCentari** [comment]
Sure I can do that. I can take off the Autodoc for now and see if it's
better just to do self surgery.

**Comment by BetaCentari** [comment]
> Any chance we can try just the self-surgery improvement without the autodoc? It does technically let people implant themselves, but might make them think twice about not getting a doctor to do it

Done.

**Comment by Bokkiewokkie** [comment]
> > Any chance we can try just the self-surgery improvement without the autodoc? It does technically let people implant themselves, but might make them think twice about not getting a doctor to do it
> 
> Done.

Please do actually remove the autodoc content from the PR aswell (I assume you've not done it yet because you were busy with other things).
Besides that I think it would be a terrible idea to have performing *all* surgeries on yourself be possible (disregarding the negative consequences of experiencing excrutiating pain for a moment).

**Comment by BetaCentari** [comment]
You are correct. I'll get on that when I get home.

**Comment by BetaCentari** [comment]
> > > Any chance we can try just the self-surgery improvement without the autodoc? It does technically let people implant themselves, but might make them think twice about not getting a doctor to do it
> > 
> > 
> > Done.
> 
> Please do actually remove the autodoc content from the PR aswell (I assume you've not done it yet because you were busy with other things). Besides that I think it would be a terrible idea to have performing _all_ surgeries on yourself be possible (disregarding the negative consequences of experiencing excrutiating pain for a moment).

I don't see why someone wouldn't be able to do surgery on themselves, no matter which one. I understand it's a bit ridiculous for someone to remove, say, their own brain but it'd be technically possible. If the autodoc isn't going to be added this PR, I'd like to be able to do any implant or surgery on myself regardless of which one it is. I can take a scrub through and remove some though, for instance, brain washing would be one that you shouldn't be able to perform on yourself but otherwise, I'd like most of them.

**Comment by Bokkiewokkie** [comment]
You'll also want to actually *keep* anything that double-defines the self-operable var being TRUE, since it means less conflicts in the beebase. And don't forget to add //NSV13 to the changes you do make!

File: code/modules/vending/medical.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 					/obj/item/storage/pill_bottle = 6,
 					/obj/item/reagent_containers/glass/bottle = 10,
 					/obj/item/healthanalyzer = 4,
-				    /obj/item/reagent_containers/spray/cleaner = 1,
+				    /obj/item/reagent_containers/spray/cleaner = 3,
```
Comment (Bokkiewokkie) [inline code/modules/vending/medical.dm:15]: 🤨

**Comment by BetaCentari** [comment]
> You'll also want to actually _keep_ anything that double-defines the self-operable var being TRUE, since it means less conflicts in the beebase. And don't forget to add //NSV13 to the changes you do make! P.S.: Also updated the PR description for you

Thank you, Bokkie. Made the requested changes.

**Comment by Karmic-Skink** [comment]
Personally, I still feel this is power gamery for solo players trying to be RoBuSt and not communicating rather than just using their words.

Also as a concept, self surgery is extremely difficult.

**Comment by BetaCentari** [comment]
> 

I want to be able to use implants again somehow, be it the autodoc or self surgery. I play almost exclusively medical and this PR addresses the two things I'm most frustrated with. I already dropped the autodoc since that was so controversial apparently, in the very least let me do it myself.

**Comment by Archemagus** [comment]
Okey may be make NOT ALL surgeries able to self-surgery?
We have Surgery called "Coronary Bypass". You can fix your own heart in your own chest? Meh
Oh, i got better one. "Organ Manipulation". Yeah - main thread in discuss. Just imagine. Surgeon, removed his own eyes. And replace them with Thermals. Or just self brain removal.

That's insane.
BUT.
I still can't see anything wrong with "Tending"-self?
Close few scars?
Some burn damage fixes?

Fair enough for... docs? In other case who will heal the Healer? (Drugs can heal, but tshhhhhhh)

**Comment by Bokkiewokkie** [comment]
After having been testmerged for a while I've decided having self-surgery like this isn't in our best interest, and that simply asking your fellow medical staff for surgery would be better. Of course you won't always have enough staff to ask for help, but you don't *need* seven implants to be the best doctor, and an important part of the game is (sometimes the lack of) cooperation.

**Pull request closed without merging**